src/components/FloatingHelper/FloatingHelper.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import pick = require('lodash/pick');
-import { bool, string, number, oneOfType, node } from 'prop-types';
+import { bool, string, number, oneOfType, node, Requireable, ValidationMap} from 'prop-types'
```

Comment:
Need also Requireable?

Reply:
Although ValidationMap "somehow" brings Requirable, I'd rather be explicit.